build/parse_test.go

```diff
@@ -32,7 +32,7 @@ func TestParse(t *testing.T) {
 		if tt.out != nil {
 			name = tt.out.Path
 		}
-		p, err := Parse(name, []byte(tt.in))
+		p, err := Parse(name, 0, []byte(tt.in))
```

Comment:
As I've mentioned, `build.Parse` is called from other projects, passing a magic number reduces readability. Perhaps it's better to create another method `ParseAuto` or a constant that's equal to 0.